test/spec/components/test/context.test.jsx

```diff
@@ -134,6 +136,45 @@ describe('<TestContext />', () => {
       expect(snippet).to.have.lengthOf(0);
       expect(img).to.have.lengthOf(1);
     });
+
+    it('renders local video', () => {
+      const context = '/testvideo.mp4';
+      const { wrapper, snippet, video } = getInstance({
+        context: JSON.stringify(context),
+        className: 'test'
+      });
+      expect(wrapper).to.have.className('test');
+      expect(snippet).to.have.lengthOf(0);
+      expect(wrapper).to.have.descendants('video');
+      expect(video).to.have.attr('src', '/testvideo.mp4');
+    });
+
+    it('renders video url with protocol', () => {
+      const context = 'http://test.url.com/testvideo.mp4';
+      const { wrapper, snippet, videoLink } = getInstance({
+        context: JSON.stringify(context),
+        className: 'test'
+      });
+      expect(wrapper).to.have.className('test');
+      expect(snippet).to.have.lengthOf(0);
+      expect(wrapper).to.have.descendants('video');
+      expect(wrapper).to.have.descendants('a.test-video-link');
+      expect(videoLink).to.have.attr('href', 'http://test.url.com/testvideo.mp4');
+      videoLink.simulate('click', { stopPropagation: noop });
```

Comment:
What's the reason for the simulated click here?

Reply:
I don't know, it was part of the img one I copied.

Reply:
Haha. Ok fair enough. I don't remember why I added that.